### PR TITLE
dropdown styling

### PR DESCRIPTION
**Pull request by archiewood**
### Description

Adds basic, consistent styling for the Dropdown component

### Checklist

- [x] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

### Chrome

CleanShot 2023-12-11 at 19 04 36@2x

### Safari

CleanShot 2023-12-11 at 19 06 04@2x

## Firefox

CleanShot 2023-12-11 at 19 06 40@2x

### CODE REVIEW

File: packages/core-components/src/lib/atoms/dropdown/Dropdown.svelte [hunk 1]
```diff
@@ -37,36 +37,40 @@
...
-		disabled={hasQuery && !$query.loaded}
-		on:change={(e) => ($inputs[name] = e.currentTarget.value)}
-	>
-		<slot />
+	{:else}
+		<select
+			disabled={hasQuery && !$query.loaded}
+			on:change={(e) => ($inputs[name] = e.currentTarget.value)}
+			class="border border-gray-300 bg-white rounded-lg p-1 mt-2 px-2 pr-5 flex flex-row items-center max-w-fit bg-transparent cursor-pointer bg-right bg-no-repeat"
+			style="background-image: url('data:image/svg+xml,%3Csvg xmlns=\'http://www.w3.org/2000/svg\' class=\'icon icon-tabler icon-tabler-chevron-down\' width=\'18\' height=\'18\' viewBox=\'0 0 24 24\' stroke-width=\'2\' stroke=\'currentColor\' fill=\'none\' stroke-linecap=\'round\' stroke-linejoin=\'round\'%3E%3Cpath stroke=\'none\' d=\'M0 0h24v24H0z\' fill=\'none\'/%3E%3Cpath d=\'M6 9l6 6l6 -6\' /%3E%3C/svg%3E');"
```
[the remark this answers is not in the document]
Comment (archiewood) [reply packages/core-components/src/lib/atoms/dropdown/Dropdown.svelte:65]: I think the 3P option would be something like https://svelte-headlessui.goss.io/docs/2.0/listbox#basic-example

**Comment by archiewood** [comment]
no idea why this mac node 20 test is failing - happening every time for a different reason

**Comment by mcrascal** [comment]
I've got conflicting icons and base styles in safari. Going to throw in some fixes. 

CleanShot 2023-12-12 at 13 43 33@2x

**Comment by mcrascal** [comment]
CleanShot 2023-12-12 at 13 51 13@2x

**Comment by mcrascal** [comment]
I missed a build step -- disregard the above 👍

File: packages/core-components/src/lib/atoms/dropdown/Dropdown.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (ItsMeBrianD) [reply packages/core-components/src/lib/atoms/dropdown/Dropdown.svelte:65]: I don't love this; there are probably alternatives but if it works then I say we run with it for now

**Review by ItsMeBrianD — APPROVED** [review]

**Pull request merged**